tests/pytest/vital_records/tasks/test_email.py

```diff
@@ -51,14 +71,27 @@ def test_handler(
         mock_inst = mocker.MagicMock(email_address="email@example.com", number_of_records=1, type=request_type)
         mock_VitalRecordsRequest.get_with_status.return_value = mock_inst
         mock_render = mocker.patch("web.vital_records.tasks.email.render_to_string", return_value="email body")
-        mock_EmailMultiAlternatives = mocker.patch("web.vital_records.tasks.email.EmailMultiAlternatives")
-        mock_email = mock_EmailMultiAlternatives.return_value
-        mock_email.send.return_value = 0
+        mock_email_office = mocker.MagicMock()
+        mock_email_requestor = mocker.MagicMock()
+        mock__create_base_email = mocker.patch(
+            "web.vital_records.tasks.email.EmailTask._create_base_email",
+            side_effect=[
+                mock_email_office,
+                mock_email_requestor,
+            ],
```

Comment:
This prompted me to read up more on [`side_effect`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.side_effect), and I learned that when it's an iterable like this, these are the return values, used one at a time and in order, if you call the mock multiple times. I didn't know this! That's cool :+1: